### PR TITLE
Update README to reflect the related changes delivered in V1.6.0

### PR DESCRIPTION
**Pull request by steven-zou**
**Update README to reflect the related changes delivered in V1.6.0**

Signed-off-by: Steven Zou <szou@vmware.com>

### CODE REVIEW

**Comment by reasonerjt** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -18,10 +18,11 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
 
 ### Features
 
+* **Cloud-native registry**: Image management and helm charts management with similar user experiences are both supported. Harbor can serve cloud-native environments like container runtimes and orchestration platform [Kubernetes](https://kubernetes.io/) as cloud-native registry.
```
Comment (jonasrosland) [inline README.md]: Small rewrite, and also making sure we follow the [CNCF Style Guide](https://github.com/cncf/foundation/blob/master/style-guide.md) with regards to the spelling of "cloud native" (no hyphen).

```
**Cloud native registry**: With support for both container images and [Helm](https://helm.sh) charts, Harbor serves as registry for cloud native environments like container runtimes and orchestration platforms.
```

File: README.md [hunk 2]
```diff
@@ -18,10 +18,11 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
...
 * **Role based access control**: Users and repositories are organized via 'projects' and a user can have different permission for images under a project.
-* **Policy based image replication**: Images can be replicated (synchronized) between multiple registry instances, with auto-retry on errors. Great for load balancing, high availability, multi-datacenter, hybrid and multi-cloud scenarios.
+* **Policy based image replication**: Images can be replicated (synchronized) between multiple registry instances based on policies with kinds of filters (repository, tag and label) supporting, with auto-retry on errors. Great for load balancing, high availability, multi-datacenter, hybrid and multi-cloud scenarios.
```
Comment (jonasrosland) [inline README.md]: Small rewrite:
```
Images can be replicated (synchronized) between multiple registry instances based on policies with multiple filters (repository, tag and label). Harbor will auto-retry to replicate if it encounters any errors. Great for load balancing, high availability, multi-datacenter, hybrid and multi-cloud scenarios.

File: README.md [hunk 3]
```diff
@@ -18,10 +18,11 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
...
 * **Vulnerability Scanning**: Harbor scans images regularly and warns users of vulnerabilities.
-* **LDAP/AD support**: Harbor integrates with existing enterprise LDAP/AD for user authentication and management.
+* **LDAP/AD support**: Harbor integrates with existing enterprise LDAP/AD for user authentication and management. Meanwhile, support importing LDAP group into Harbor and assigning proper project roles to it.
```
Comment (jonasrosland) [inline README.md]: Small rewrite:
```
Harbor integrates with existing enterprise LDAP/AD for user authentication and management, and supports importing LDAP groups into Harbor and assigning proper project roles to them.
```

**Comment by jonasrosland** [comment]
LGTM!

**Pull request merged**